experiments/src/rai_experiments/models/pretrained.py

```diff
@@ -0,0 +1,96 @@
+from functools import partial
+
+import pooch
+from typing_extensions import Literal
+
+__all__ = ["load_model"]
+
+
+_pre_trained_manager = pooch.create(
+    path=pooch.os_cache("rai-experiments"),
+    base_url="https://github.com/mit-ll-responsible-ai/data_hosting/releases/download/madrylab_models/",
+    registry={
+        "mitll_cifar_l2_1_0.pt": "md5:738f109685dfc55f7227dc6dc473379e",
+        "mitll_cifar_nat.pt": "md5:34e15ff183735933e13f41ad27f755b4",
+        "mitll_restricted_imagenet_l2_3_0.pt": "md5:db37145acfd8bdf7ef7c05f4d261c0d9",
+        "mitll_imagenet_l2_3_0.pt": "md5:37728e5c9c47684c0edd373e9d80ac9b",
+    },
+)
+
+
+def load_model(
+    model_name: Literal[
+        "mitll_cifar_l2_1_0.pt",
+        "mitll_cifar_nat.pt",
+        "mitll_imagenet_l2_3_0.pt",
```

Comment:
Could we also have this function load a natural ImageNet model (i.e., download torchvision's resnet50 with pretrained weights)?

Reply:
(I went ahead and did this and pushed a commit with this added... feel free to modify it though)